### DISCUSSION TITLE
What is this?

### DISCUSSION THREAD

**Discussion by Vtuber-i-love** [netless-io/flat#1721] [Q&A] [answered]
**What is this? How do you authenticate?
I don't understand, so tell me in Japanese!**
---
![D176647C-AF45-4118-B25D-E52018DC9967_1_201_a](https://user-images.githubusercontent.com/115625874/195314960-90d82a91-787c-47e5-9e3e-3106503040a3.jpeg)

**Comment by BlackHole1** [comment, marked as answer]
Because of Chinese policy requirements, we must require users to bind their cell phone number. 😔
You can enter your cell phone number, and click "Send code", after a while, you will receive a text message, at this time you enter the 6 digits in the text message into the application and that's it.

**Reply by BlackHole1** [reply]
I'm planning to deploy the flat app overseas, at which point we won't require users to be tied to a phone number.

**Reply by Vtuber-i-love** [reply]
I'm using a software called Flat with the same score, but it was confusing with the same software name. What is the genre of this software?

**Reply by BlackHole1** [reply]
This is free online classroom software.
